presto-main/src/main/resources/webapp/assets/query.js

```diff
@@ -892,6 +892,32 @@ var QueryDetail = React.createClass({
     },
     renderFailureInfo: function() {
         var query = this.state.query;
+        var taskRow = "";
+        var hostRow = "";
+        if (query.failureTask != null) {
+            taskRow = (
+            <tr>
+                <td className="info-title">
+                    Failed Task
```

Comment:
Rename the "Error Information" section header (below in the source code) to be "Failure Information".

@raghavsethi went through some designs and end up with this:
<img width="505" alt="screen shot 2017-01-24 at 12 21 28 pm" src="https://cloud.githubusercontent.com/assets/9230/22265281/45cff752-e230-11e6-86e4-2bf145b72cda.png">